service/src/main/java/bio/terra/cbas/common/MetricsUtil.java

```diff
@@ -161,6 +173,10 @@ public static void registerAllViews() {
                     25000.0,
                     50000.0)));
 
+    Aggregation recordsPerRequestDistribution =
+        Aggregation.Distribution.create(
+            BucketBoundaries.create(Arrays.asList(0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 100.0)));
```

Comment:
Can/should these be integers instead of `.0`s?

Reply:
it seems like `BucketBoundaries` wants a list of Doubles: https://javadoc.io/doc/io.opencensus/opencensus-api/0.21.0/io/opencensus/stats/BucketBoundaries.html